.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help the OpenTX team to improve OpenTX
+
+---
+*Before submitting,  search the open issue if there is already a report of the bug*
```

Comment:
And search closed ones too... seems like we get quite a few dupes of issues already fixed but not yet released.  In fact I'd probably emphasize searching closed issues in particular.